### PR TITLE
nautilus ansible fragments

### PR DESCRIPTION
**Pull request by hmaheswa**
Signed-off-by: Hemanth Maheswarla <hmaheswa@hmaheswa.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by hmaheswa** [comment]
jenkins downstream test run logs for tier-0_deploy_containerized_ceph workflow
https://ceph-downstream-jenkins-csb-storage.apps.ocp4.prod.psi.redhat.com/job/fragments/15/consoleFull

File: fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+tests:
```
Comment (psathyan) [inline fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml:1]: Can we revisit this file?

File: fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          ceph_conf_overrides:
+            client:
+              rgw crypt require ssl: false
+              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+            global:
+              mon_max_pg_per_osd: 1024
+              osd_default_pool_size: 2
+              osd_pool_default_pg_num: 64
+              osd_pool_default_pgp_num: 64
```
Comment (psathyan) [inline fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml:15]: These configurations can be depreciated.

File: fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            mon:
+              mon_allow_pool_delete: true
+          ceph_origin: distro
+          ceph_repository: rhcs
+          ceph_stable: true
+          ceph_stable_release: nautilus
```
Comment (psathyan) [inline fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml:21]: This is specific for using upstream configurations.

File: fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+              pgs: "8"
+          copy_admin_key: true
+          dashboard_admin_password: p@ssw0rd
+          dashboard_admin_user: admin
+          dashboard_enabled: true
+          fetch_directory: ~/fetch
+          grafana_admin_password: p@ssw0rd
+          grafana_admin_user: admin
+          grafana_container_image: "registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4"
+          journal_size: 1024
```
Comment (psathyan) [inline fragments/install/ceph-ansible/deploy_cluster_alertmanager_container_image_override.yaml:39]: do we need to have this setting

File: fragments/teardown/ceph_cluster_purge_with_playbook_command.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+tests:
```
Comment (psathyan) [inline fragments/teardown/ceph_cluster_purge_with_playbook_command.yaml:1]: Why do we have two purge fragments

File: fragments/teardown/ceph_ansible_purge.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+tests:
```
Comment (psathyan) [inline fragments/teardown/ceph_ansible_purge.yaml:1]: IMO, we should have `fragments/teardown/ceph-ansible/purge.yaml` like the way use in install

**Comment by psathyan** [comment]
@hmaheswa I am closing this PR for now and we can rework on it once the priority and feature has stabilized. Thanks for your contribution.

**Pull request closed without merging**